kolibri/plugins/facility_management/assets/src/views/class-learner-enroll-page.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+
+  <div>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'classLearnerEnrollPage',
```

Comment:
"class learner enroll" page sounds like a bag of words that humans wouldn't say.

On the mocks, the header of this page is "Enroll learners into 'Class'". Could we name it something close to that like "EnrollLearnersPage" or "LearnerEnrollmentPage"?

Reply:
I agree, I was sort of shoehorning the word "class" in to denote that it's a class-related page within the `facility_management` plugin